server/package.json

```diff
@@ -65,7 +65,8 @@
     "populate_db:debug": "nodemon --watch ../data -e csv --exec babel-node --inspect src/populate_db.js",
     "prebuild": "rm -rf transpiled_build",
     "build": "babel src/ --out-dir transpiled_build --copy-files --ignore node_modules",
-    "mongod": "sh ../scripts/dev_scripts/local_mongod.sh -p 27017"
+    "mongod": "sh ../scripts/dev_scripts/local_mongod.sh -p 27017",
+    "update_outage_info": "gsutil cp ./outage_msg.json gs://ib-outage-bucket/"
```

Comment:
Oh so if we're updating it using this, why did you decide to put `outage_msg.json` in gitignore?

Reply:
I was thinking that only the person making the update to the bucket would need to have the file on their system